src/main/java/io/meeds/qa/ui/pages/SpaceHomePage.java

```diff
@@ -1780,7 +1780,7 @@ private ElementFacade kudosButtonNumberToCommentActivityElement() {
   }
 
   private ElementFacade kudosButtonToCommentActivityElement() {
-    return findByXPathOrCSS("//div[contains(@class,'white border-radius')][1]//div[contains(@class,'v-list flex')]//div[contains(@class,'d-inline-flex')][1]//div[@role='button']//span[@class='v-btn__content'][last()]");
+    return findByXPathOrCSS("//div[contains(@class,'unread-activity')][1]//div[contains(@class,'v-list flex')]//*[contains(@id, 'ActivityCommment_comment')]//div[contains(@class,'d-inline-flex')][1]//div[@role='button']//span[@class='v-btn__content'][last()]");
```

Comment:
We should have something similar to what was made in method `getActivityFooterButton`

```
    return getCommentFooterButton(comment, "KudosActivity");
```